rdt/hyper_transformer.py

```diff
@@ -653,25 +548,25 @@ def _fit_field_transformer(self, data, field, transformer):
         """
         if transformer is None:
             self._add_field_to_set(field, self._fitted_fields)
-            self._transformers_tree[field]['transformer'] = None
-            self._transformers_tree[field]['outputs'] = [field]
+            if field not in self._output_columns:
+                self._output_columns.append(field)
```

Comment:
why do we need this?

Reply:
I realized I misread this code, so it would have been fine to leave this in although I don't think it will ever see the same output column more than once